docs/import/manage_disaster.js

```diff
@@ -559,57 +553,65 @@ function onNonDamageAssetSelect(
 const lastSelectedAsset = new Map();
 
 /**
- * Checks the given asset for the given columns and prints a result message.
+ * Verifies an asset exists and has the expected columns.
  * @param {string} asset
  * @param {string} type values from the first index of each entry in {@code
  *     scoreAssetTypes}
  * @param {string} state e.g. 'WA'
  * @param {Array<string>} expectedColumns
- * @return {?Promise<unknown>} returns null if column checking wasn't needed.
+ * @return {Promise<void>} returns null if column checking wasn't needed.
```

Comment:
do we ever return null now? Might also want to rephrase to mention existence checking.

Reply:
Still do when we don't have an asset to check (set to None)